test/features/standup.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Standup-module
+
+    Scenario: get list of standups
+        Given make a GET request to "/standup"
+        Then the response status code should be 200
+        And the response should be "[]"
+
+    Scenario: add a new standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then the response status code should be 201
+        And the response should contains:
+            | discordUser      | "eddiehubber"              |
+            | yesterdayMessage | "yesterday I did this"     |
+            | todayMessage     | "Today I'll do this"       |
+            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | id               | 666                        |
+
+
+    Scenario: search existing Standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then  make a GET request to "/standup/search?discordUser=eddiehubber"
+        Then the response status code should be 200
+        And  the response in item "0" should contains:
+            | discordUser      | "eddiehubber"              |
+            | yesterdayMessage | "yesterday I did this"     |
+            | todayMessage     | "Today I'll do this"       |
+            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | id               | 666                        |
+
+    Scenario: search non-existing Standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then  make a GET request to "/standup/search?discordUser=hubber"
+        Then the response status code should be 200
+        And  the response should be "[]"
+
+    Scenario: provide no Search Context
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then  make a GET request to "/standup/search"
+        Then the response status code should be 400
+        And  the response should contains:
+            | statusCode | 400                             |
+            | message    | "Please Provice Search Context" |
+
+
+
+    Scenario: add an empty standup
```

Comment:
`standup` here is lowercase and in other places it has a capital. We should try to be consistent. I think having everything lower case in the `Scenario` would make it easier for everyone to be consistent

For example https://github.com/EddieHubCommunity/api/pull/54/files#diff-757cd4aa33cb116bb7e4ebeef9a619042a9b5f4fe6acae2b75bdfbba93b74ca1R45

Reply:
Yes, I definitly see this issue. 
I'll fix this.